### PR TITLE
Add functions to change channel

### PR DESCRIPTION
**Pull request by christopher77**
**This pull request includes:**
- Add renderChannel function, here we get the data from the localstorage and set it in a list in the channel.html file

- Add changeChannel function which recive the name of a channel and put it in the channel-header element
solved #5

### CODE REVIEW

**Comment by condef5** [comment]
Update the description please, the description should detail all the features you did :).

File: public/js/main.js [hunk 1]
```diff
@@ -1,27 +1,24 @@
-console.log("🚀");
+// Data to test functions
+localStorage.setItem(
+  "channels",
+  JSON.stringify(["general", "XXX", "heavy", "top-secret"])
```
Comment (test0n3) [inline public/js/main.js]: Quite the heavy channels. \m/

Comment (DiegoCuevas) [inline public/js/main.js]: please change that names.

File: public/js/main.js [hunk 2]
```diff
@@ -1,27 +1,24 @@
...
+console.log("🚀");
 
-function createChannel(nameChannel) {
-  if (!compareName(nameChannel))
-    channels.push({ id: new Date().getTime(), name: nameChannel });
-  else console.log("The channel already exist!");
+function renderChannel() {
+  const channels = JSON.parse(localStorage.getItem("channels"));
+  $listChannel.innerHTML = channels.reduce((html, tittle) => {
+    const command = `<li onclick = "changeChannel('${tittle}')"> # ${tittle}</li>`;
```
Comment (condef5) [inline public/js/main.js]: Hey, you could rename this function `changeChannel` by `handleChangeChannel`

Comment (christopher77) [reply public/js/main.js]: thank you, I did not know that.

Comment (condef5) [reply public/js/main.js]: You could rename variable `tittle` by `channel`, it's more for convention issues and code readability :)

File: public/js/main.js (hunk 1 above)
Comment (christopher77) [reply public/js/main.js]: yes, I forgot to change it.

File: public/js/main.js (hunk 2 above)
Comment (christopher77) [reply public/js/main.js]: you're right, I will do it

**Pull request merged**